composer.json

```diff
@@ -17,6 +17,8 @@
   },
   "php-ext": {
     "extension-name": "mongodb",
+    "override-download-url-method": "pre-packaged-source",
```

Comment:
Needs php/pie#187 to be merged before this will work

Reply:
Above is merged, release will be very soon, just finishing off the uninstall feature in PIE and will make a next release :)